compliance_checker/ioos.py

```diff
@@ -950,6 +953,56 @@ def _check_gts_ingest_val(self, val):
 
         return isinstance(val, str) and val.lower() in {"true", "false"}
 
+    def check_vertical_coordinates(self, ds):
+        '''
+        Check that vertical units (corresponding to axis "Z") are a unit
+        equivalent to one of "meter", "inch", "foot", "yard", "US_survey_foot",
+        or "fathom".  Check that the vertical coordinate variable "positive"
+        attribute is either "up" or "down".  Note that unlike the CF version
+        of this check, pressure units are not accepted and length units are
+        constrained to the aforementioned set instead of accepting any valid
+        UDUNITS length unit.
+
+        :param netCDF4.Dataset ds: An open netCDF dataset
+        :rtype: list
+        :return: List of results
+        '''
+        ret_val = []
+        z_variables = cfutil.get_z_variables(ds)
+        for name in z_variables:
+            variable = ds.variables[name]
+            standard_name = getattr(variable, 'standard_name', None)
```

Comment:
Maybe can skip this as it's not used anywhere.

Reply:
Agreed

Reply:
Oops, meant to do that.